breaker/breaker.go

```diff
@@ -0,0 +1,29 @@
+package breaker
+
+import (
+	"fmt"
+	"github.com/afex/hystrix-go/hystrix"
+	"github.com/rcrowley/go-metrics"
+	"net/http"
+	"time"
+)
+
+func Do(name string, url string) (resp *http.Response, err error) {
+	return DoWithFallback(name, url, nil)
+}
+
+func DoWithFallback(name string, url string, f func(error) error) (resp *http.Response, err error) {
+	err = hystrix.Do(name, func() error {
+		start := time.Now()
+		if resp, err = Get(url); err == nil {
```

Comment:
Right now it's not needed. I guess they should be in separate packages. I'll consider that. Thanks